builtin/rebase.c

```diff
@@ -353,7 +353,8 @@ static int run_specific_rebase(struct rebase_options *opts)
 #define GIT_REFLOG_ACTION_ENVIRONMENT "GIT_REFLOG_ACTION"
```

Comment:
Also, this commit message is the appropriate place to make an argument for reusing `reset_head()` instead of implementing a separate function `move_to_branch()` (as the Unix shell script did).